webapp/src/lib/api.js

```diff
@@ -30,15 +30,27 @@ class API {
     return this.request('get', '/projects', {})
   }
 
-  postOutbidNotification(email, parcelStateIds) {
+  postSignedOutbidNotification(message, signature, parcelStateIds) {
     return this.request('post', '/outbidNotification', {
-      email,
+      message,
+      signature,
       parcelStateIds: parcelStateIds.join(';')
     })
   }
 
-  deleteOutbidNotification(email) {
-    return this.request('delete', '/outbidNotification', { email })
+  postOutbidNotification(address, parcelStateIds) {
+    return this.request('post', '/outbidNotification', {
+      address,
+      parcelStateIds: parcelStateIds.join(';')
+    })
+  }
+
+  deleteOutbidNotifications(address) {
```

Comment:
What if we remove this endpoint which allows anyone to unsubscribe you and send the signed message when we click on the unsubscribe button in the sidebar?

Reply:
Yes, you are right, saw that too! but it's more difficult as we have to sign the message again when adding new bids to validate you are the one registering the parcels. Maybe keeping a signature in localStorage?

For not allowing registration on parcels you haven't bid we could add that validation I think on the registerParcelNotifications() method. Right?